pkg/cmd/server/api/validation/master.go

```diff
@@ -209,6 +215,25 @@ func ValidateEtcdStorageConfig(config api.EtcdStorageConfig) fielderrors.Validat
 	return allErrs
 }
 
+func ValidateStorageVersionLevels(level string, knownAPILevels, deadAPILevels []string, name string) ValidationResults {
+	validationResults := ValidationResults{}
+
+	if len(level) == 0 {
+		validationResults.AddErrors(fielderrors.NewFieldRequired(name))
+	}
+
+	deadLevels := sets.NewString(deadAPILevels...)
+	knownLevels := sets.NewString(knownAPILevels...)
+	if deadLevels.Has(level) {
+		validationResults.AddWarnings(fielderrors.NewFieldInvalid(name, level, "unsupported level"))
```

Comment:
@smarterclayton made a strong statement disallowing usage: https://github.com/openshift/origin/pull/4992#issuecomment-148128210 .  That would make these errors.